ksql-common/src/test/java/io/confluent/ksql/util/KsqlConfigTest.java

```diff
@@ -138,7 +139,16 @@ public void shouldSetPrefixedStreamsConfigProperties() {
   }
 
   @Test
-  public void shouldCleanStreamsProperties() {
+  public void shouldSetMonitoringInterceptorConfigProperties() {
+    final KsqlConfig ksqlConfig = new KsqlConfig(Collections.singletonMap(
+        "confluent.monitoring.interceptor.topic", "foo"));
+    final Object result
+        = ksqlConfig.getKsqlStreamConfigProps().get("confluent.monitoring.interceptor.topic");
```

Comment:
Would be good to use the actual monitoring interceptor config. It is something like `producer.interceptor.classes` and `consumer.interceptor.classes`.

Reply:
I can add a case for that, though It should be covered by the producer/consumer cases anyway.